lambdas/get-addresses/src/services/config-service.ts

```diff
@@ -0,0 +1,37 @@
+import { SSMClient, GetParameterCommand } from "@aws-sdk/client-ssm";
+
+export class ConfigService {
+    constructor(private ssmClient: SSMClient) {}
+
+    private parameterPrefix = process.env.AWS_STACK_NAME || "";
+    readonly config: { addressLookupStorageTableName: string | undefined } = {
+        addressLookupStorageTableName: undefined,
+    };
+
+    public async init(): Promise<void[]> {
+        // how do handle the parameter names
+        const getParameterAddressQueryTable = this.getParameter(
+            `${this.parameterPrefix}/GetAddressQueryTable`,
+            "addressLookupStorageTableName",
+        );
+
+        return Promise.all([getParameterAddressQueryTable]);
```

Comment:
Future-proofing this, if we ever add more variables we can promise.all them to keep `init` speed high.